docs/source/api_reference/threshold.rst

```diff
@@ -71,13 +71,15 @@ Examples
     height_max = np.max(z)
 
     # Hide parts of the mesh
-    threshold_mesh = Threshold(mesh, input='height', min=height_min, max=height_max)
+    threshold_mesh = Threshold(mesh, input='height', min=height_min, max=height_max, range=(height_min, height_max))
```

Comment:
Unfortunately I did not find a way to make `range` optional here

omitting `range=` (or passing invalid type) will raise 
```python
---------------------------------------------------------------------------
KeyError                                  Traceback (most recent call last)
~\Anaconda3\envs\ipygany_dev\lib\site-packages\traitlets\traitlets.py in get(self, obj, cls)
    534         try:
--> 535             value = obj._trait_values[self.name]
    536         except KeyError:

KeyError: 'range'

During handling of the above exception, another exception occurred:

# ....

TraitError: The 'range' trait of a Threshold instance expected a tuple, not the NoneType None.
```

Reply:
You created the trait with `Tuple(CFloat(0.), CFloat(0.))`, I suppose this doesn't work, you might need to do something like: `Tuple((0, 0))`

Reply:
that works, thanks